tests/run_all_tests.sh

```diff
@@ -43,33 +43,41 @@ handle_errors () {
 }
 
 # Run battery of core FLAX API tests.
-pytest tests -n auto $PYTEST_OPTS
-
-# validate types
-if [[ "$OSTYPE" == "darwin"* ]]; then
-  echo "Pytype is currently not working on MacOS, see https://github.com/google/pytype/issues/661"
-else
-  pytype flax/
-fi
+pytest -n 4 tests $PYTEST_OPTS
 
 # Per-example tests.
-if [[ $ALL_EXAMPLES == 'true' ]]; then
```

Comment:
Please also remove `ALL_EXAMPLES=false`, help message and `--all` switch above.

Reply:
Good idea.